packages/aws-lambda/test/integration_test/test_definition.js

```diff
@@ -1064,40 +1092,6 @@ function registerTests(handlerDefinitionPath) {
     });
   });
 
-  describe('API Gateway/EUM - Server-Timing array header already present', function () {
```

Comment:
_issue(blocking):_ I disagree with this deletion, I think we should restore this test. 

> FYI: Removed the array case, because we already test adding response.multiValueHeaders. No need to test that twice.

I think those are two different things. The "normal" test for `response.multiValueHeaders` we only test capturing headers that the Lambda handler sets on its own, that is, in particular this code: https://github.com/instana/nodejs/blob/feat/lambda-payload2.0/packages/aws-lambda/src/capture_headers.js#L34-L43

For the Server-Timing header it is different. We manipulate this header in the response deliberately to inject the trace ID for website monitoring there. And this needs to be handled differently depending on whether the lambda handler already sets this header or not, and whether it has been set by the Lambda handler as a multi value header or a single header.

This test was for this part of the code: https://github.com/instana/nodejs/blob/feat/lambda-payload2.0/packages/aws-lambda/src/process_result.js#L71-L74